docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,252 @@
+       <privilege>http://tizen.org/privilege/datasharing</privilege>
+    </privileges>
+    ```
+
+2.  Using the methods and properties of the `Tizen.Nlp` namespace, include it in your application:
+
+    ```
+    using Tizen.Nlp;
+    ```
+<a name="wordtokenize"></a>
+## Receive WordTokenize Tokens
+
+To receive the tokens from sentence:
+
+1.  Construct NaturalLanguageProcess on OnCreate(), and connect the NLP service on init of app:
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the WordTokenizeAsync(string msg) in a async task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.WordTokenizeAsync(msg);
+        WordTokenizeResult wr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release the resource of Nlp:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+<a name="postag"></a>
+## Receive PosTag Tokens and Tags
+
+To receive the tokens and tags from sentence:
+
+1.  Construct NaturalLanguageProcess on OnCreate(), and connect the NLP service on init of app:
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the PosTagAsync(string msg) in a async task method:
+
+    ```
+    public async Task OnPosButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.PosTagAsync(msg);
+        PosTagResult pr = await ret;
+        ..
+    }
+    ```
+
+3.  When nlp object is no longer needed, call Dispose() to release the resource of NLP:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+<a name="nechunk"></a>
+## Receive Named Entity Tokens and Tags
+
+To receive the tokens and tags info from sentence:
+
+1.  Construct NaturalLanguageProcess on OnCreate(), and connect the NLP service on init of app:
+
+    ```
+    protected override void OnCreate
+    {
+        ..
+        nlp = new NaturalLanguageProcess();
+        Task task = nlp.Connect();
+        ..
+    }
+    ```
+
+2.  Call the NamedEntityRecognitionAsync(string msg) in a async task method:
+
+    ```
+    public async Task OnTokenButtonPressedAsync(string msg)
+    {
+        ..
+        var ret = nlp.NamedEntityRecognitionAsync(msg);
+        NamedEntityRecognitionResult nr = await ret;
+        ..
+    }
+    ```
+
+3.  When it is no longer needed, call Dispose() to release the resource of NLP:
+
+    ```
+    protected override void OnTerminate()
+        {
+         ..
+         nlp.Dispose();
+         ..
+        }
+    ```
+
+
+<a name="lemmatize"></a>
+## Receive Lemmatize ActualWord
```

Comment:
@divyk , I'm not sure with this title.
Lemmatize? Lemmatized? Which one is proper?
ActualWord? Actual Word?
Would you please check?

Reply:
@divyk `[LemmatizeResult](#lemmatize)` on line 23 is trying to go to #lemmatize,
which is assigned as `<a name="lemmatize"></a>` on line 169.
Even the title string on line 170 is updated, as it doesn't  effect `<a name="lemmatize"></a>` on line 169, we don't need to upldate the link.

@zg2nets, would you please check "Receive Lemmatized Actual Word" and apply it, if the meaning is not changed?